router/router.nginx.conf.in

```diff
@@ -59,6 +59,18 @@ server {
     listen [::]:80;
 }
 
+server {
```

Comment:
If we don't add a build.yaml entry that creates a service called `batch2`, all the routers in PRs will fail to start (and the main router will fail to start if there's no service in the default namespace).